selftests/functional/test_lv_utils.py

```diff
@@ -30,7 +33,10 @@ def tearDown(self):
         for vg_name in self.vgs:
             lv_utils.vg_remove(vg_name)
 
-    @unittest.skipIf(process.system("modinfo scsi_debug", ignore_status=True),
+    @unittest.skipIf(sys.platform.startswith('darwin'),
+                     'macOS does not support LVM')
+    @unittest.skipIf(process.system("modinfo scsi_debug", shell=True,
```

Comment:
if shell=False, the modinfo check will throw an OSError instead of the regular CmdError exception, making the ignore_status check useless.